examples/event_handling/pick_event_demo.py

```diff
@@ -100,8 +101,10 @@ def onpick1(event):
             print('onpick1 text:', text.get_text())
 
     fig.canvas.mpl_connect('pick_event', onpick1)
+    plt.show()
 
```

Comment:
Please use two blank lines between top-level functions. flake8 will be complaining about this.